### PR TITLE
PS-4764 fix documenting ft_query_extra_word_chars variable

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 1]
```diff
@@ -54,3 +53,38 @@ System Variables
...
+Punctuation Marks in Full-Text Search
+=====================================
+
+By default, full text search is unable to find words with various punctuation
+characters in boolean search mode, although those characters are
+indexed with ngram parser. A new variable :variable:`ft_query_extra_word_chars`
+was introduced in |Percona Server| :rn:`5.7.21-20` to solve this issue.
+
+When it's enabled, all the non-whitespace symbols are considered to be
+word symbols by FTS query parser, except for the boolean search syntax
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/innodb_fts_improvements.rst:68]: link to upstream variable specifying the syntax symbols

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 2]
```diff
@@ -54,3 +53,38 @@ System Variables
...
+symbols. The latter ones are also considered to be word symbols inside
+double quote. This only applies for the query tokenizer, and the
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/innodb_fts_improvements.rst]: inside double quoteS ?

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/flexibility/innodb_fts_improvements.rst (hunk 1 above)
Comment (fiowro) [reply doc/source/flexibility/innodb_fts_improvements.rst:68]: done

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 3]
```diff
@@ -54,3 +53,38 @@ System Variables
...
+=====================================
+
+By default, full text search is unable to find words with various punctuation
+characters in boolean search mode, although those characters are
+indexed with ngram parser. A new variable :variable:`ft_query_extra_word_chars`
+was introduced in |Percona Server| :rn:`5.7.21-20` to solve this issue.
+
+When it's enabled, all the non-whitespace symbols are considered to be
+word symbols by FTS query parser, except for the boolean search syntax
+symbols (which are specified by `ft_boolean_syntas <https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_ft_boolean_syntax>`_ variable). The latter ones are also considered to be word symbols inside
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/innodb_fts_improvements.rst]: "ft_boolean_syntas"

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/flexibility/innodb_fts_improvements.rst (hunk 3 above)
Comment (fiowro) [reply doc/source/flexibility/innodb_fts_improvements.rst]: fixed

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 4]
```diff
@@ -54,3 +53,38 @@ System Variables
...
+.. variable:: ft_query_extra_word_chars
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Session, Global
+  :dyn: Yes
+  :vartype: Boolean
+  :default: ``OFF``
+
+When enabled, this variable will make all non-witespace symbols (including
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/innodb_fts_improvements.rst]: whitespace

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**